swagger2locustio/__main__.py

```diff
@@ -14,14 +14,27 @@ def main():
     """Launching function"""
 
     parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-    parser.add_argument("--swagger-file", help="path to swagger file", required=True, type=Path)
+    parser.add_argument("-f", "--swagger-file", help="path to swagger file", required=True, type=Path)
     parser.add_argument(
-        "--results-path", help="path to store locustfile.py", required=False, default=Path("generated"), type=Path,
+        "-r",
```

Comment:
here short arg came before long, will it be correct to parse?

Reply:
It seems alright as swagger2locustio --help returns such lines:

-f SWAGGER_FILE, --swagger-file SWAGGER_FILE
                         path to swagger file (default: None)
-r RESULTS_PATH, --results-path RESULTS_PATH
                         path to store locustfile.py (default: generated)